database/seed.js

```diff
@@ -0,0 +1,10 @@
+const db = require('./index.js');
+const Restaurant = require('./Restaurant.js').Restaurant;
+const restaurantData = require('./Restaurant.js').mockData;
+ 
+
+const insertData = () => {
+  Restaurant.create(restaurantData);
```

Comment:
Currently there is no way for us to know if this succeeded or failed without looking at the database